zebra/zebra_router.c

```diff
@@ -278,6 +284,12 @@ void zebra_router_init(bool asic_offload, bool notify_on_ack,
 
 	zrouter.nhg_keep = ZEBRA_DEFAULT_NHG_KEEP_TIMER;
 
+	/*Init V6 RA batching stuffs*/
+	srand(time(NULL));
```

Comment:
please remove srand() and just use the frr_weak_random function call where necessary.